components/badge.css

```diff
@@ -0,0 +1,115 @@
+.badge-container {
+  box-sizing: border-box;
+  width: fit-content;
+  height: fit-content;
+  position: relative;
+}
+
+.badge {
+  border: 1px solid white;
```

Comment:
you can try using HEX or RGB method to set color property

Reply:
Yes, fixed.